lib/features/popup-menu/PopupMenu.js

```diff
@@ -237,11 +237,7 @@ PopupMenu.prototype._onClosed = function() {
 
 PopupMenu.prototype._createContainer = function(config) {
 
-  let parent = config && config.parent || 'body';
-
-  if (typeof parent === 'string') {
-    parent = document.querySelector(parent);
-  }
+  let parent = document.querySelector('.djs-container');
```

Comment:
We'd need to retrieve it from the [diagram container](https://github.com/bpmn-io/diagram-js/blob/develop/lib/core/Canvas.js#L491) (cf. original implementation) or we risk finding a adjacent diagram container, i.e. if multiple diagram-js instances are embedded on the same page.